src/components/mixins/Leaderboard.js

```diff
@@ -36,8 +41,16 @@ module.exports = {
   loadLeaderboard: function(type) {
     this.setState({ isLoading: true });
 
+    var groupValue = this.props.groupValue;
+    if (this.props.groupValues && this.props.groupValues.length > 0) {
+      groupValue = paramJoin(this.props.groupValues, '&groupValue[]');
```

Comment:
Whether it's the most ideal way of doing it is questionable (there's neater ways of encoding arrays in a URL). But the endpoint currently accepts multiple group values by repeating the parameter for each group value to pass as a single string, so `paramJoin()` will make that happen, and that's why the test expects a single string instead of an array.